unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/ECSRuntime/ComponentsGroup/ECSComponentsGroupOfTwo.cs

```diff
@@ -0,0 +1,57 @@
+using System.Collections.Generic;
+using DCL.Controllers;
+using DCL.Models;
+
+namespace DCL.ECSRuntime
+{
+    public class ECSComponentsGroupOfTwo<T1, T2> : IECSComponentsGroup, IECSReadOnlyComponentsGroupOfTwo<T1, T2>
+    {
+        private readonly IECSComponent component1;
+        private readonly IECSComponent component2;
+        private readonly List<ECSComponentsGroupOfTwoData<T1, T2>> list = new List<ECSComponentsGroupOfTwoData<T1, T2>>();
+
+        IReadOnlyList<ECSComponentsGroupOfTwoData<T1, T2>> IECSReadOnlyComponentsGroupOfTwo<T1, T2>.group => list;
+
+        public ECSComponentsGroupOfTwo(IECSComponent component1, IECSComponent component2)
+        {
+            this.component1 = component1;
+            this.component2 = component2;
```

Comment:
I would add some null Asserts here, it will gives indsight for unexpected errors while avoiding pitfalls in production.

Reply:
done